### PR TITLE
Forgot to add them to the readme

### PR DESCRIPTION
**Pull request by chrismainey**
Also added the licencing from Wikipedia.  Not sure of requirements here, but Beatles copyright holders are known to go after people, so wanted to cover my back.

### CODE REVIEW

**Comment by johnmackintosh** [comment]
Hi Chris, as discussed, there has been a bit of crossover as I updated the package inbetween your earlier PR and this one. 
I've removed all album images as part of these updates. 
If you are able to update the fork with the latest version, we can look at updating the palettes if need be. 
Melloncollie already added to the package readme and website , so don't worry about not having added them earlier.

**Comment by chrismainey** [comment]
I've sorted my mess in the readme out.  I'm not sure about junking the siamese dream one.  Four colour is very close to swift, but 10 colour is ok.  Happy if you want to junk it though.

**Comment by johnmackintosh** [comment]
TLDR, Siamese dream is / will be IN
 I see that the ten colour Siamese dream palette is in already, but undocumented on the readme. Some how it's missing from rock_palettes (my fault). It's only the abbey road palettes I  intended to leave out for now.
Will sort it all later on.
Thanks  Chris 👍
